bootstrap-salt.sh

```diff
@@ -4563,15 +4563,16 @@ install_freebsd_git_deps() {
     fi
     echodebug "Finished patching"
 
+    # Set _SALT_ETC_DIR to ports default
+    _SALT_ETC_DIR=${BS_SALT_ETC_DIR:-/usr/local/etc/salt}
+    # We also need to redefine the PKI directory
+    _PKI_DIR=${_SALT_ETC_DIR}/pki
+
     # Let's trigger config_salt()
     if [ "$_TEMP_CONFIG_DIR" = "null" ]; then
         _TEMP_CONFIG_DIR="${_SALT_GIT_CHECKOUT_DIR}/conf/"
-        CONFIG_SALT_FUNC="config_salt"
+        CONFIG_SALT_FUNC="config_freebsd_salt"
```

Comment:
Do we need to switch this here? Since all that does is call the same line that is set above: `_SALT_ETC_DIR=${BS_SALT_ETC_DIR:-/usr/local/etc/salt}` and then call config_salt?

I also don't see anywhere else that we're calling that function, so we might want to just remove it. What do you think? I might be missing something.